oregon/templates/site.html

```diff
@@ -37,7 +37,12 @@
     #myCarousel {
         margin-top: 10px;
     }
-
+    .place-label {
+    fill:#8C001A;
+    font-size:14;
```

Comment:
I'm getting a CSS error here, and it's just ignoring the font size.  You have to specify the meaning of the number: `pt`, `px`, `em`, etc.